testsuite/fuzzer/fuzz.sh

```diff
@@ -16,11 +16,13 @@ function error() {
 }
 
 function cargo_fuzz() {
-    rustup install nightly
+    # Workaround for build failures on oss-fuzz caused by dependencie incompatible with 1.78.0
+    # Owner: @zi0Black
+    rustup install nightly-2024-01-01
```

Comment:
we may just leave this as a "version control" mechanism to avoid cases where nightly breaks - which will again